api/src/pcapi/core/offerers/tasks.py

```diff
@@ -43,11 +43,24 @@ def check_offerer_siren_task(payload: CheckOffererSirenRequest) -> None:
         tag = offerers_models.OffererTag.query.filter(offerers_models.OffererTag.name == "siren-caduc").one()
 
         with transaction():
-            db.session.add(offerers_models.OffererTagMapping(offererId=offerer.id, tagId=tag.id))
-            history_api.add_action(
-                history_models.ActionType.INFO_MODIFIED,
-                None,
-                offerer=offerer,
-                comment="La structure est détectée comme inactive via l'API Sirene (INSEE)",
-                modified_info={"tags": {"new_info": tag.label}},
-            )
+            if offerer.validationStatus == offerer.validationStatus.PENDING:
```

Comment:
On doit rejeter les `PENDING` mais aussi les `NEW`, ces deux états sont en attente de validation
```suggestion
            if offerer.isWaitingForValidation:
```